samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,163 @@
+.. _direction_finding_connectionless_tx:
+
+Bluetooth: Direction Finding Connectionless Beacon
+##################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+A simple application demonstrating the BLE Direction Finding CTE Broadcaster functionality
+by transmission of Constant Tone Extension with periodic advertising PDUs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix to use it in Angle of Departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix or any other antenna matrix.
+
+Configuration
+*************
+
+|config|
+
+The sample supports two Direction Finding modes:
+
+* Angle of Arrival
+* Angle of Departure
+
+By default both modes are available in the sample application.
+
+Angle of Arrival mode
+=====================
+
+To build this sample with Angle of Arrival mode only, set :makevar:`OVERLAY_CONFIG`
+to :file:`overlay-aoa.conf`.
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars`
+in the Zephyr documentation.
+
+Antenna matrix configuration
+============================
+
+To use this sample with Angle of Departure enabled, additional GPIOS configuration
+is required to enable control of the antenna array. Example of such configuration
+is provided in :file:`nrf52833dk_nrf52833.overlay`.
+
+:file:`nrf52833dk_nrf52833.overlay` is a device tree overlay that provides information
+about which GPIOS should be used by Radio peripheral to switch between antenna
+patches during CTE transmission in AoD mode. At least two GPIOs must be provided
+to enable antenna switching. GPIOS are used by Radio peripheral in order following
+indices of :code:`dfegpio#-gpios` properties. The order is important because it
+affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
```

Comment:
> The GPIOs are used by the Radio peripheral in order following indices of :code:dfegpio#-gpios properties.
There may be a number of `dfegpio#-gpios` properties in devicetree e.g. dfegpio1-gpios, dfegpio2-gpios, dfegpio3-gpios. 
I wanted to inform end used that order of GPIOs used by Radio peripheral is provided by dfegpio#-gpios. 

For:
dfegpio1-gpios = pin 3
dfegpio2-gpios = pin 2
dfegpio3-gpios = pin 33
Radio uses: GPIOs = {3, 2, 33} 
For:
dfegpio1-gpios = pin 33
dfegpio2-gpios = pin 2
dfegpio3-gpios = pin 3
Radio uses: GPIOs = {33, 2, 3}.

That order matters, because each GPIO is mapped bits in an antenna pattern described later.

:code: tag is leftover from Zephyr rst.

Reply:
:code: removed and used instead `` text ``

Reply:
Ok, so how about:

The GPIOs are used by the Radio peripheral in order given by the ``dfegpio#-gpios`` properties.

Reply:
Also remember to add the other suggested changes too :)

Reply:
Added missing changes. Also I've used suggested sentence about GPIOs order.

Reply:
What is wrong with this one?

Reply:
The rest fixed by https://github.com/nrfconnect/sdk-nrf/pull/4044#discussion_r584645035